src/PoolManager.sol

```diff
@@ -26,6 +26,54 @@ import {Extsload} from "./Extsload.sol";
 
 /// @notice Holds the state for all pools
```

Comment:
I feel like this line should go directly above
```
contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claims, Extsload {
```

? thoughts?

Reply:
agreed fixed